db/models/amenities.js

```diff
@@ -18,8 +18,9 @@ let amenitiesMethod = {
     return Amenities.findOne({ _id: id }).exec();
   },
   create: ( one ) => {
+    console.log('create:', one);
```

Comment:
Does this console log still need to be in the code?

Reply:
i will update latest one